spinn_utilities/camel_case_config_parser.py

```diff
@@ -0,0 +1,76 @@
+from ConfigParser import RawConfigParser
+from distutils.util import strtobool
+
+
+class CamelCaseConfigParser(RawConfigParser):
+
+    def optionxform(self, optionstr):
+        lower = optionstr.lower()
+        return lower.replace("_", "")
+
+    def __init__(self, defaults=None, none_marker="None"):
+        RawConfigParser.__init__(self, defaults)
+        self._none_marker = none_marker
```

Comment:
From Donal's config branch: add a _read_files property to store the read config files.  The read_a_config (and check_config as that is only used by read_a_config) from conf_loader.py can then also be moved in here, and can be extended to add the file read to the _read_files property.